TL improvement (#377)
07th-mod/higurashi-console-arcs: Update/omot_022.txt

			NULL, "\"Nfu fu fu!!", Line_WaitForInput);
	ModPlayVoiceLS(4, 11, "ps3/s14/11/220700266", 256, TRUE);
	OutputLine(NULL, "　そりゃまぁ、私だってにっくき園崎の娘を牢屋にぶち込んで、あの皺だらけのばーさんと小憎らしい姐御に一泡吹かせてやりたい、とは思いますけどね？」",

> 			NULL, " Well, I want to throw a daughter of the Sonozakis in jail and incur the wrath of that wrinkly old hag and your bratty sister, to be honest.\"", Line_Normal);

Reviewer: ```suggestion
			NULL, " Well to be honest, I'd like to throw a daughter of the Sonozakis in jail and incur the wrath of that wrinkly old hag and your bratty sister.\"", Line_Normal);
```

	ClearMessage();

//大石rvS14/11/220700267.「あの古手家頭首を守るために命がけで来てるんだ、それくらいのことはさせてもらいますよ」
	if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#a59da9>大石</color>", NULL, "<color=#a59da9>Ooishi</color>", NULL, Line_ContinueAfterTyping); }
	ModPlayVoiceLS(4, 11, "ps3/s14/11/220700267", 256, TRUE);
	OutputLine(NULL, "「あの古手家頭首を守るために命がけで来てるんだ、それくらいのことはさせてもらいますよ」",

> 			NULL, "\"But you're risking your life to protect the Furude family head, so of course I have to treat you like now.\"", Line_Normal);

Reviewer: ```suggestion
			NULL, "\"But you're risking your life to protect the Furude family head, so of course I'll play along.\"", Line_Normal);
```

//r大石の話だと、フィルムに偽装される形で様々な書類が隠されていたそうだ。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "大石の話だと、フィルムに偽装される形で様々な書類が隠されていたそうだ。",

> 			NULL, "According to Ooishi, those various documents were disguised as films.", Line_Normal);

Reviewer: ```suggestion
			NULL, "According to Ooishi, various documents were hidden in there, disguised as rolls of film.", Line_Normal);
```

			NULL, "\"Meep~. Then if I hear anything, I'll kick you straight out of the bed.", Line_WaitForInput);
	ModPlayVoiceLS(4, 5, "ps3/s14/05/220500220", 256, TRUE);
	OutputLine(NULL, "どちらが少なかったか競争なのですよ～」",

> 			NULL, " Let's start a contest. the one with less snores wins~.\"", Line_Normal);

Reviewer: ```suggestion
			NULL, " Let's make it a contest. Whoever snores less wins~.\"", Line_Normal);
```

//r可愛いいびきをかいて眠るのは、沙都子のほうが圧倒的に多い。ほぼ間違いなく私の勝ちだろうが、…今回は勝ちを譲ってあげようかな。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "可愛いいびきをかいて眠るのは、沙都子のほうが圧倒的に多い。ほぼ間違いなく私の勝ちだろうが、…今回は勝ちを譲ってあげようかな。",

> 			NULL, "In Satoko's adorable sleep, she makes far, far more snores than I. So it's pretty much a guaranteed win for me... but I'll let her have the victory this time.", Line_Normal);

Reviewer: ```suggestion
			NULL, "Satoko snores far, far more often than I do in her adorable sleep. So it's pretty much a guaranteed win for me... but I'll let her have the victory this time.", Line_Normal);
```

	ClearMessage();

//r…入江の手配で、私と沙都子はひそかに空いた診察室に移された。私は沙都子のベッドの下にマットを敷かれ、医薬品の箱の陰に隠れる格好で寝かされている。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "…入江の手配で、私と沙都子はひそかに空いた診察室に移された。私は沙都子のベッドの下にマットを敷かれ、医薬品の箱の陰に隠れる格好で寝かされている。",
			NULL, "...Irie arranged for Satoko and me to be moved to a vacant exam room. I'm lying on a mat under Satoko's bed, hidden behind a box of medical supplies.", Line_Normal);
	ClearMessage();

//r襲撃を受けた場所に、まさかとどまってはいまい。…詩音のアイデアだった。そして、夕方ごろ入江がお魎の診察に向かう時に沙都子と二人で車に同乗して、園崎家の地下部屋に隠れる手はずとなっている。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	if (GetGlobalFlag(GADVMode)) { OutputLine(NULL, "", NULL, "<size=-2>", Line_Continue); }
	OutputLine(NULL, "襲撃を受けた場所に、まさかとどまってはいまい。…詩音のアイデアだった。そして、夕方ごろ入江がお魎の診察に向かう時に沙都子と二人で車に同乗して、園崎家の地下部屋に隠れる手はずとなっている。",

> 			NULL, "It's hard to predict we're simply staying put at the scene of the attack. ...It was Shion's idea. And when Irie heads over to give Oryou a medical exam later this evening, Satoko and I will go for a ride in his car and then end up hidden away in the basement of the Sonozaki house.", Line_Normal);

Reviewer: ```suggestion
			NULL, "They'd never expect we're simply staying put at the scene of the attack. ...It was Shion's idea. And when Irie heads over to give Oryou a medical exam later this evening, Satoko and I will go for a ride in his car and then end up hidden away in the basement of the Sonozaki house.", Line_Normal);
```

			NULL, "\"No, please listen.", Line_WaitForInput);
	ModPlayVoiceLS(4, 4, "ps3/s14/04/220400075", 256, TRUE);
	OutputLine(NULL, "私ずっと眠っている間、あの時のことを何度も夢に見ました。",

> 			NULL, " While I was sleeping, I dreamed about back then, many times.", Line_WaitForInput);

Reviewer: ```suggestion
			NULL, " I've had dreams about back then, many times.", Line_WaitForInput);
```

	if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#fcdb77>沙都子</color>", NULL, "<color=#fcdb77>Satoko</color>", NULL, Line_ContinueAfterTyping); }
	ModPlayVoiceLS(4, 4, "ps3/s14/04/220400083", 256, TRUE);
	OutputLine(NULL, "でも、…ほれた弱み、とでも申し上げるのでしょうか。",

> 			NULL, "But... in short, it says love is blind, right?", Line_Normal);

Reviewer: ```suggestion
			NULL, "But... in short, they say love is blind, right?", Line_Normal);
```

//r友達をたくさん呼べるように、最新のＴＶゲームを。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "友達をたくさん呼べるように、最新のＴＶゲームを。",

> 			NULL, "Purchasing a latest video game console to have them call over lots of friends.", Line_Normal);

Reviewer: ```suggestion
			NULL, "Buying the latest video game console so they can invite lots of friends over.", Line_Normal);
```

	if (GetLocalFlag(LConsoleArc)==10) {
		ModPlayVoiceLS(4, 0, "ps2/00/222900029", 540, TRUE);
		OutputLine(NULL, "「あ…あんたに取り調べの決定権はないはずだ！",
				NULL, "\"Y... you don't have the right to make decisions regarding the interrogation!", Line_WaitForInput);
		ModPlayVoiceLS(4, 0, "ps2/00/222900030", 540, TRUE);
		OutputLine(NULL, "　警部より下の人間に、容疑者を拘留できるとでも思ってるのか？！」",

> 				NULL, " Do you think inferior people to a chief inspector can exercise the power of detention?!\"", Line_Normal);

Reviewer: ```suggestion
				NULL, " Do you think people ranking below chief inspector can detain a suspect?!\"", Line_Normal);
```

	} else {
		ModPlayVoiceLS(4, 0, "ps3/s14/00/222900029", 256, TRUE);
		OutputLine(NULL, "「あ…あんたに取り調べの決定権はないはずだ！",
				NULL, "\"Y... you don't have the right to make decisions regarding the interrogation!", Line_WaitForInput);
		ModPlayVoiceLS(4, 0, "ps3/s14/00/222900030", 256, TRUE);
		OutputLine(NULL, "　警部より下の人間に、容疑者を拘留できるとでも思ってるのか？！」",

> 				NULL, " Do you think inferior people to a chief inspector can exercise the power of detention?!\"", Line_Normal);

Reviewer: ```suggestion
				NULL, " Do you think people ranking below chief inspector can detain a suspect?!\"", Line_Normal);
```

			NULL, "\"Call him back?", Line_WaitForInput);
	ModPlayVoiceLS(4, 11, "ps3/s14/11/220700354", 256, TRUE);
	OutputLine(NULL, "　…いっときますけど、彼は気弱だが信念は飛び切り強い男だ。",

> 			NULL, " ...Just so you know, He may be a timid man, but he still has very strong convictions.", Line_WaitForInput);

Reviewer: ```suggestion
			NULL, " ...Just so you know, he may be a timid man, but he still has very strong convictions.", Line_WaitForInput);
```

		if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#797d8a>赤坂</color>", NULL, "<color=#797d8a>Akasaka</color>", NULL, Line_ContinueAfterTyping); }
		ModPlayVoiceLS(4, 13, "ps3/s14/13/vt4bc_akasaka1078", 256, TRUE);
		OutputLine(NULL, "「とはいえ私たちは、仮にも次期首相候補と目されていた実力者が、こんなリスクの高いプロジェクトに関わるわけがないと当初は思っていました」",
				NULL, "\"But at first we thought the powerful man, who was future candidate for prime minister, couldn't get involved in such a highrisk project.\"", Line_Normal);
		ClearMessage();

	//赤坂rvS14/13/VT4Bc_akasaka1079.「しかし、やつはそれに飽き足らず、プロジェクトを立ち上げた亡き政商の金脈をひとり占めにして、さらに権力を拡大しようと目論んでいた…」
		if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#797d8a>赤坂</color>", NULL, "<color=#797d8a>Akasaka</color>", NULL, Line_ContinueAfterTyping); }
		ModPlayVoiceLS(4, 13, "ps3/s14/13/vt4bc_akasaka1079", 256, TRUE);
		OutputLine(NULL, "「しかし、やつはそれに飽き足らず、プロジェクトを立ち上げた亡き政商の金脈をひとり占めにして、さらに権力を拡大しようと目論んでいた…」",

> 				NULL, "\"But unsatisfied with that, he was trying to expand his power by monopolizing the funds left by the late influence peddler, who launched the project...\"", Line_Normal);

Reviewer: ```suggestion
				NULL, "\"But unsatisfied with that, he was trying to expand his power by monopolizing the funds left by the late political merchant who launched the project...\"", Line_Normal);
```

				NULL, "\"Those ambitions ended up an empty dream leading to being arrested.", Line_WaitForInput);
		ModPlayVoiceLS(4, 13, "ps3/s14/13/vt4bc_akasaka1081", 256, TRUE);
		OutputLine(NULL, "そして、鷹野三四を動かそうとしていた連中は、敵対勢力であるそいつを排除するのが目的だったようです」",

> 				NULL, " And those who were manipulating Miyo Takano were doing so with the intention of eliminating that politian who belongs to an opposition party.\"", Line_Normal);

Reviewer: ```suggestion
				NULL, " And those who were manipulating Miyo Takano were doing so with the intention of eliminating that politician in an opposition party.\"", Line_Normal);
```

		if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#797d8a>赤坂</color>", NULL, "<color=#797d8a>Akasaka</color>", NULL, Line_ContinueAfterTyping); }
		ModPlayVoiceLS(4, 13, "ps3/s14/13/vt4bc_akasaka1084", 256, TRUE);
		OutputLine(NULL, "「近いうちに連中は、凶行を起こそうと起こすまいと、自分たちは今回の企てに無関係だと彼女たちに通達するでしょう」",

> 				NULL, "\"In the near future, they'll profess that they have nothing to do with her conspiracies, whether or not she cease any acts of violence.\"", Line_Normal);

Reviewer: ```suggestion
				NULL, "\"In the near future, whether she ceases her acts of violence or not, they'll profess that they have nothing to do with her conspiracies.\"", Line_Normal);
```

		ClearMessage();

	//赤坂rvS14/13/VT4Bc_akasaka1086.「公安としても、結果として外事的に問題のあった政治家を排除できたので、彼らの以後の保身活動については黙認してやるつもりです。kvS14/13/VT4Bc_akasaka1087.…あくまで今回は、ですが」
		if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#797d8a>赤坂</color>", NULL, "<color=#797d8a>Akasaka</color>", NULL, Line_ContinueAfterTyping); }
		ModPlayVoiceLS(4, 13, "ps3/s14/13/vt4bc_akasaka1086", 256, TRUE);
		OutputLine(NULL, "「公安としても、結果として外事的に問題のあった政治家を排除できたので、彼らの以後の保身活動については黙認してやるつもりです。",
				NULL, "\"In terms of public security, we were able to eliminate the politicians with problematic foreign relations, so we'll tacitly accept their future self-preservation tactics.", Line_WaitForInput);
		ModPlayVoiceLS(4, 13, "ps3/s14/13/vt4bc_akasaka1087", 256, TRUE);
		OutputLine(NULL, "…あくまで今回は、ですが」",
				NULL, " ...At least for this time, though.\"", Line_Normal);
		ClearMessage();
	}

//r赤坂はそう言って、皮肉めいた笑みを浮かべる。…おそらく公安と同じような仕事をする人間なんだろうが、そのやり方に気にいらないものがあったようだ。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "赤坂はそう言って、皮肉めいた笑みを浮かべる。…おそらく公安と同じような仕事をする人間なんだろうが、そのやり方に気にいらないものがあったようだ。",

> 			NULL, "Akasaka cynically smiled as he said that. ...They were probably performing a job similar to the Public Security, but there was something he didn't like them.", Line_Normal);

Reviewer: ```suggestion
			NULL, "Akasaka cynically smiled as he said that. ...They were probably performing a job similar to Public Security, but there was something he didn't like about them.", Line_Normal);
```
